src/controllers/cart.controller.js

```diff
@@ -0,0 +1,88 @@
+import { createCart, updateCart } from '../services/cart.service';
+import { findProductById } from '../services/product.service';
+import { getCartProducts } from '../services/cart.service';
+import culculateProductTotal from '../utils/cart';
+
+const addToCart = async (req, res) => {
+  try {
+    const userId = req.user.id;
+    let cart = req.cart;
+
+    let { productId } = req.body;
+    let productQuantity = req.body.productQuantity || 1;
```

Comment:
can you remove unwanted lines please, see if it can be done in a single line

ex: const { productId, cart } = req.body;